eng/sendtohelix.proj

```diff
@@ -85,6 +85,19 @@
           Outputs="$(TestArchiveRuntimeFile)"
           Condition="'$(TargetGroup)' != 'AllConfigurations'">
 
+    <!-- Add global tools -->
+    <ItemGroup Condition="'$(TargetsNetCoreApp)' == 'true'">
+      <TestArchiveRuntimeDependencies Include="$(GlobalToolsDir)**\*.*" />
```

Comment:
Should this be included as part of the targets input? If the tools change and we update them locally or something, it should re zip the folder, right?

Reply:
That directory will never change but to be more correct we should add it to the inputs. Thanks, will add it.